core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -132,7 +132,105 @@ describe('Overview', () => {
             expect(jazzServices_po.getYearVerify().getText()).toEqual('YEAR');
             browser.sleep(5000);
             jazzServices_po.getServiceFromAsset().click();
-      });      
+      });
+
+      it('Verify METRICS Navigation for API' , () => {
+            // Navigation to services
+            browser.wait(EC.visibilityOf(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            //To Navigate to the particular service and verifying the Page
+            jazzServices_po.getAwsServiceName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+            browser.sleep(5000);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+            jazzServices_po.getRefresh().click();
+            browser.sleep(2000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdName()), timeOutHigh);
+            jazzServices_po.getProdName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
+            browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
+            browser.driver.switchTo().activeElement();
+            browser.sleep(5000);
+            jazzServices_po.getMetrices().click();
+            browser.driver.sleep(5000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getDeploymentStatus()), timeOutHigh);
+            jazzServices_po.getDeploymentStatus().click();
+            expect(jazzServices_po.getTestAPI().getText()).toEqual('TEST API');
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getTestAPI()), timeOutHigh);
+            jazzServices_po.getTestAPI().click();
+            browser.sleep(15000);
+            browser.getAllWindowHandles().then(function(handles){
+                  browser.switchTo().window(handles[1]).then(function(){
+                        expect(jazzServices_po.getAPIGET().getText()).toEqual('GET');
+                        jazzServices_po.getAPIGET().click();
+                        browser.sleep(5000);
```

Comment:
line 166 can these sleep be removed? Remove all sleep in the code that are not required

Reply:
removed the sleep here in this place. but most of the places it require as after click event, it mostly unable to find the element therefore we are giving sleep to wait for it.